shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -254,6 +256,10 @@ bool FlutterWindowsEngine::RunWithEntrypoint(const char* entrypoint) {
     auto host = static_cast<FlutterWindowsEngine*>(user_data);
     return host->HandlePlatformMessage(engine_message);
   };
+  args.vsync_callback = [](void* user_data, intptr_t baton) -> void {
+    auto host = static_cast<FlutterWindowsEngine*>(user_data);
+    return host->OnVsync(baton);
```

Comment:
Nit: superfluous return in a method that returns void.

Reply:
done